CLAUDE.md

```
@@ -203,13 +203,27 @@ Configure simulations via constants at top of `src/bin/simulations/main.rs`:
+`types`:
+
+```rust
+struct Slot(u64);                          // src/types/slot.rs
+struct Stake(u64);                         // src/types/stake.rs
+struct ValidatorIndex(u64);                // src/types/validator_index.rs (NOT `ValidatorId`)
+struct SliceIndex(u64);                    // src/types/slice_index.rs
+struct Fraction { numerator, denominator } // src/types/fraction.rs
+```
+
+The only bare aliases live in `src/lib.rs` / `src/crypto/merkle.rs`:
 ```rust
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)